### DISCUSSION TITLE
Add row disposition on clickable README summary

### DISCUSSION THREAD

**Discussion by ANFADEV** [npm/feedback#800] [Website]
Clickable summaries are not aligned in a row, it should be done with just a few css lines

In Github:
![image](https://user-images.githubusercontent.com/32341740/198736838-d738e763-a1a0-46da-afe4-fa5da5270b97.png)

On npmjs.com:
![image](https://user-images.githubusercontent.com/32341740/198736891-2ad779ad-554a-4204-af7d-6806b1810435.png)

**Comment by ljharb** [comment]
Just curious, what HTML do you have there inside your `<summary>`?
